### ISSUE TITLE
should aria-controls be conditionally required on combobox?

### ISSUE THREAD

**Issue by pkra** [w3c/aria#2604]
Extracted from @smhigley's comment https://github.com/w3c/aria/pull/2583/files#r2286395664 

> aria-controls should be conditionally required on combobox, right? That is, my understanding is if it's expanded it's required, and not if collapsed. Should this be kept, since while it's not absolutely required, it may still need a fallback defined for the expanded state?

(Though I'm not sure we currently have a good way to express such a conditional requirement in the spec.)

**Comment by pkra** [comment]
It seems relevant that the question was raised on a change to the table "Fallback values for missing required attributes".

**Comment by giacomo-petri** [comment]
FYI:
the aria-controls requirement for expanded comboboxes was introduced in 2020 with this PR https://github.com/w3c/aria/pull/1335/files and later removed in March 2024 with this PR https://github.com/w3c/aria/pull/2134
